modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -106,6 +114,15 @@ const RichText = ({
 			editor.config.contentsLanguage = currentEditingLocale.localeId;
 			editor.setData(currentInternalValue);
 		}
+
+		setCKEditor5Config({
```

Comment:
shouldn't this be wrapped into a FF check as well? we don't want to leak behaviors between 2 editors. (same for ckeditor4 part)